lib/index.js

```diff
@@ -21,9 +41,21 @@ bedrock.events.on('bedrock.init', () => {
 
 bedrock.events.on('bedrock.start', () => {
   const {config: {'did-io': cfg}} = bedrock;
-
+  const didKeyDriver = didMethodKey.driver(cfg.methods.key);
+  const {keyTypes} = cfg.methods.key;
+  for(const keyType in keyTypes) {
+    const verificationSuite = SUPPORTED_KEY_TYPES.get(keyType);
+    if(!verificationSuite) {
```

Comment:
So these aren't "verificationSuites" (and we should drop that terminology), so I just called them handlers here for lack of a better term:

```suggestion
    const handler = SUPPORTED_KEY_TYPES.get(keyType);
    if(!handler) {
```

Reply:
Updated [d55315f](https://github.com/digitalbazaar/bedrock-did-io/pull/16/commits/d55315f93d25c1b0c1930eccc31f223ce6998e9b)